sdcm/sct_config.py

```diff
@@ -1015,6 +1016,18 @@ def __init__(self):
             self['target_upgrade_version'] = get_branch_version(new_scylla_repo)
         LOGGER.info(self.dump_config())
 
+        # 9) validate seeds number
```

Comment:
these checks should be in `sdcm.sct_config.SCTConfiguration#verify_configuration`

Reply:
moved